src/components/AnnouncementsPage/Announcements.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+
+export default function AnnouncementsPage() {
+	const theme = useTheme();
+
+	return (
+		<>
+			<Typography variant='h4' component='h1'
+				style={{ fontWeight: theme.typography.fontWeightBold,
+					paddingTop: theme.spacing(37), // initial value 4
+					paddingBottom: theme.spacing(37), // initial value 4
```

Comment:
I can see that you used this to vertically center the text on the screen and also make text fill up the entire screen (so the footer doesn't look awkward in the middle of the screen). Screen sizes differ, however, and this results in the page sometimes taking up too much height and sometimes still taking up too little height. 

I just made new fix that makes all pages take up a minimum height of the size of the screen (regardless of how much content is on the page) so you don't need to have this large padding anymore. Feel free to just use the original `theme.spacing(4)` padding. It's not a big deal if the text is not vertically centered.